README.md

```diff
@@ -39,6 +39,7 @@ If you want a quick way to test or run code, checkout the [Playground](https://p
 * BBEdit: [bbedit-pony](https://github.com/TheMue/bbedit-pony)
 * Micro: [micro-pony-plugin](https://github.com/Theodus/micro-pony-plugin)
 * Nano: [pony.nanorc file](https://github.com/serialhex/nano-highlight/blob/master/pony.nanorc)
+* Kate: update syntax definition file: Settings -> Configure Kate -> Open/Save -> Modes & Filetypes -> Download Highlighting Files
```

Comment:
Should this include a link to the syntax definition file to use?

Reply:
Something like this?

[pony.xml](https://phabricator.kde.org/source/syntax-highlighting/browse/master/data/syntax/pony.xml): native in the latest version, otherwise follow the instructions for [The Highlight Definition XML Format](https://docs.kde.org/stable5/en/applications/katepart/highlight.html) or update syntax definition file: Settings -> Configure Kate -> Open/Save -> Modes & Filetypes -> Download Highlighting Files.

Or keep only the first part?

Reply:
Ah, so the syntax highlighter rules are already bundled with the latest version of the editor?

Reply:
Yes, but the distributions are not yet up to date. And some like Manjaro use a previous version of pony.xml that has a bug that can block the parser in an infinite loop. So, the easiest way is to select the file to update via the dedicated button (which is well hidden).